custom-bundle.Dockerfile

```diff
@@ -27,6 +27,7 @@ FROM quay.io/openstack-k8s-operators/mariadb-operator-bundle:latest as mariadb-b
 FROM quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle:latest as rabbitmq-bundle
 FROM quay.io/openstack-k8s-operators/placement-operator-bundle:latest as placement-bundle
 FROM quay.io/openstack-k8s-operators/glance-operator-bundle:latest as glance-bundle
+FROM quay.io/openstack-k8s-operators/cinder-operator-bundle:latest as cinder-bundle
```

Comment:
We'll also need to `COPY` from `cinder-bundle` and then pass its CSV to the `csv-merger` below

Reply:
I did this in a follow up. Thanks.